.editorconfig

```diff
@@ -38,6 +38,9 @@ indent_style = tab
 charset = utf-8
 insert_final_newline = true
 
+[Python.Runtime.csproj]
+charset = latin1
```

Comment:
Like I commented on the other PR, I don't get this one. I actually think it's wrong and just works accidentally because the file is ASCII-only right now.

Reply:
Removed